web/styles/zulip.css

```diff
@@ -2446,7 +2448,7 @@ select.invite-as {
 }
 
 @media (width < $md_min) {
-    .column-left {
+    .app-main .column-left {
         display: none;
```

Comment:
This `display: none` is being removed from the navbar `column-left` part; I assume that's probably a noop because a more specific rule would determine if that's show anyway?

Reply:
I think this `display: none` is indeed lost, but has no practical effect, because the navbar logo area is hidden behind the rest of the navbar thanks to `position: absolute`. But it's probably better for it not to be removed?

Reply:
OK but the `column-right`  variable was just a noop as that was already `display: none`.